package.json

```diff
@@ -1,18 +1,27 @@
 {
   "name": "metrics_webapp_project",
   "version": "0.1.0",
+  "homepage": "https://imranhossainemi.github.io/metrics_webapp_project",
```

Comment:
- [ ] As suggested by the previous reviewer, kindly fix your deployment link :thinking: actually the link is deployed properly but the problem is in your package.json file :fearful: now kindly remove this **homepage** key-value pair in line 4 of package.json and try to deploy your app again Inshallah it will work :blush: